File: app/src/main/java/cat/mvmike/minimalcalendarwidget/infrastructure/receiver/InstanceChangeReceiver.java
// Copyright (c) 2016, Miquel Martí <miquelmarti111@gmail.com>
// See LICENSE for licensing information

package cat.mvmike.minimalcalendarwidget.infrastructure.receiver;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

import cat.mvmike.minimalcalendarwidget.application.MonthWidget;

public class InstanceChangeReceiver extends BroadcastReceiver {

    @Override
    public void onReceive(final Context context, final Intent intent) {
        MonthWidget.forceRedraw(context);
    }
}

File: app/src/main/kotlin/cat/mvmike/minimalcalendarwidget/infrastructure/receiver/InstanceChangeReceiver.kt
// Copyright (c) 2016, Miquel Martí <miquelmarti111@gmail.com>
// See LICENSE for licensing information
package cat.mvmike.minimalcalendarwidget.infrastructure.receiver

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import cat.mvmike.minimalcalendarwidget.MonthWidget

class InstanceChangeReceiver : BroadcastReceiver() {

    override fun onReceive(context: Context, intent: Intent) {
        MonthWidget.forceRedraw(context)
    }
}
